web-services/client/src/main/java/datawave/webservice/query/metric/QueryMetric.java

```diff
@@ -633,9 +652,27 @@ public int getFieldNumber(String name) {
             fieldMap.put("plan", 34);
             fieldMap.put("loginTime", 35);
             fieldMap.put("predictions", 36);
+            fieldMap.put("version", 37);
         }
     };
     
+    private static String getVersionFromProperties() {
+        String returnStr = "";
+        try {
+            final Properties props = new Properties();
+            String inputPath = QueryMetric.class.getClassLoader().getResource("version.properties").getPath();
+            props.load(new FileInputStream(inputPath));
+            returnStr = props.getProperty("currentVersion");
+        } catch (IOException e) {
+            log.error("Unable to find a version from resources properties file: ", e);
+            returnStr = "";
+        } catch (NullPointerException e) {
+            log.error("Null pointer encountered attempting to draw version.properties resource file: ", e);
```

Comment:
See https://stackoverflow.com/a/15146399.  It'd be best to do a check for null rather than catch this.

Reply:
Tweaked this in the latest commit, thanks